docs/src/main/asciidoc/hibernate-orm-panache-guide.adoc

```diff
@@ -381,6 +381,37 @@ Make sure to wrap methods modifying your database (e.g. `entity.persist()`) with
 CDI bean method `@Transactional` will do that for you and make that method a transaction boundary. We recommend doing 
 so at your application entry point boundaries like your REST endpoint controllers.
 
+== Lock management
+
+Panache don't provides direct support to lock the database, but you can easily do it by injecting the `EntityManager` in your entity (or `PanacheRepistory`) and creating a specific method that will use the entity manager to lock the entity after retrieval. The entity manager can also be retrieved via `Panache.getEntityManager()`.
+
+The following exemple contains a `findByIdForUpdate` method that find the entity by primary key then lock it. The lock will generate a `SELECT ... FOR UPDATE` query (the same principle can be used for other kind of `find*` methods):
```

Comment:
s/exemple/example/
s/find/finds/
s/lock/locks/
s/kind/kinds/

Reply:
done, sorry, english is not my native language.